src/View/nav.html.twig

```diff
@@ -1,52 +1,66 @@
-<nav role='navigation'>
+<header class='top-bar'>
 	<div class="main">
 		<a href="" {# Enter link here #}>
 			<img src="../assets/images/logook.png" class="logo" alt="logo"></a>
-		<div class="search-box" id="search">
-			<img src="../assets/images/magglass.png">
-			<input type="text" class="input-search" placeholder="Type to Search...">
-			<button class="submit-search">Submit</button>
-		</div>
-		<div class="menu" id="icons" onclick="searchHide();">
-			<a href="" {# Enter link here #}><img src="../assets/images/Vector.png"></a>
-			<a href="" {# Enter link here #}><img src="../assets/images/carnet.png"></a>
-			<a href="" {# Enter link here #}><img src="../assets/images/panier.png"></a>
-			<a href="" {# Enter link here #}><img src="../assets/images/compte.png"></a>
-		</div>
-		<!-- Burger Button -->
+	</div>
+	<div class="search-box" id="search">
+		<img src="../assets/images/magglass.png">
+		<input type="text" class="input-search" placeholder="Type to Search...">
+		<button class="submit-search">Go!</button>
+	</div>
+	<ul class="menu" id="icons" onclick="searchHide();"></li>
+	<li>
+		<a href="" {# Enter link here #}><img src="../assets/images/Vector.png"></a>
+	</li>
+	<li>
+		<a href="" {# Enter link here #}><img src="../assets/images/carnet.png"></a>
+	</li>
+	<li>
+		<a href="" {# Enter link here #}><img src="../assets/images/panier.png"></a>
+	</li>
+	<li>
+		<a href="" {# Enter link here #}><img src="../assets/images/compte.png"></a>
+	</li>
+</ul>
+<!-- Burger Button -->
+<ul class="burger-button">
+	<li>
 		<button class="hamburger">
 			<div class="bar"></div>
 		</button>
-	</div>
-</nav>
-<!-- Quick access categories -->
-<div class="quick-access">
-	<ul>
-		<li>
-			<a href="" {# Enter link here #}><img src="../assets/images/drawer.png"><p>Meubles</p>
-			</a>
-		</li>
-		<div class="line"></div>
-		<li>
-			<a href="" {# Enter link here #}><img src="../assets/images/lights.png"><p>Luminaires</p>
-			</a>
-		</li>
-		<div class="line"></div>
-		<li>
-			<a href="" {# Enter link here #}><img src="../assets/images/decorations.png"><p>Décorations</p>
-			</a>
-		</li>
-		<div class="line"></div>
-		<li>
-			<a href="" {# Enter link here #}><img src="../assets/images/appliances.png"><p>Électroménager</p>
-			</a>
-		</li>
-	</ul>
-</div>
-<!-- Mobile nav menu box -->
-<nav class="mobile-nav">
-	<a href="" {# Enter link here #}><img src="../assets/images/Vector.png"></a>
-	<a href="" {# Enter link here #}><img src="../assets/images/carnet.png"></a>
-	<a href="" {# Enter link here #}><img src="../assets/images/panier.png"></a>
-	<a href="" {# Enter link here #}><img src="../assets/images/compte.png"></a>
-</nav>
+	</li>
+</ul></header><nav><!-- Quick access categories --><ul class="quick-access">
+	<li>
+		<a href="" {# Enter link here #}><img src="../assets/images/drawer.png"><p>Meubles</p>
+		</a>
+	</li>
+	<div class="line"></div>
```

Comment:
should be a li tag or if it is just for "display" behavior, it should not be present here but inside the li targeted